src/main/java/uk/gov/hmcts/divorce/document/content/BailiffNotApprovedOrderContent.java

```diff
@@ -52,34 +44,27 @@
 
         final Map<String, Object> templateContent = new HashMap<>();
 
-        templateContent.put(CCD_CASE_REFERENCE, caseData.formatCaseRef(ccdCaseReference));
-        templateContent.put(DATE, LocalDate.now(clock).format(DATE_TIME_FORMATTER));
-
-        templateContent.put(PETITIONER_FULL_NAME, caseData.getApplication().getMarriageDetails().getApplicant1Name());
-        templateContent.put(RESPONDENT_FULL_NAME, caseData.getApplication().getMarriageDetails().getApplicant2Name());
-
         var alternativeService = caseData.getAlternativeService();
+
         templateContent.put(REFUSAL_REASON, alternativeService.getServiceApplicationRefusalReason());
         templateContent.put(SERVICE_APPLICATION_RECEIVED_DATE,
             alternativeService.getReceivedServiceApplicationDate().format(DATE_TIME_FORMATTER));
+        templateContent.put(CCD_CASE_REFERENCE, caseData.formatCaseRef(ccdCaseReference));
+        templateContent.put(DATE, LocalDate.now(clock).format(DATE_TIME_FORMATTER));
+        templateContent.put(PETITIONER_FULL_NAME,
+            String.format("%s %s", caseData.getApplicant1().getFirstName(), caseData.getApplicant1().getLastName()));
+        templateContent.put(RESPONDENT_FULL_NAME,
+            String.format("%s %s", caseData.getApplicant2().getFirstName(), caseData.getApplicant2().getLastName()));
         templateContent.put(PARTNER, commonContent.getPartner(caseData, caseData.getApplicant2()));
+        templateContent.put(THE_APPLICATION, caseData.getDivorceOrDissolution().isDivorce()
+            ? DIVORCE_APPLICATION : APPLICATION_TO_END_THE_CIVIL_PARTNERSHIP);
 
         var ctscContactDetails = CtscContactDetails
             .builder()
-            .poBox(poBox)
-            .town(town)
-            .postcode(postcode)
+            .emailAddress(email)
```

Comment:
should the email be dynamic for divorce/CP ?

Reply:
confirmed this from Michael earlier that it will be the static email as mentioned in the template https://docs.google.com/document/d/1-PDaz4xihVoGSH_pGREHChwVqLty6TahjU6IFgoYx8s/edit